docs/running-rules.md

```diff
@@ -9,42 +9,33 @@ import MoreHelp from "/src/components/MoreHelp"
 - Local rules:
-  - Create and run [ephemeral rules](#ephemeral-rules) in the command line.
-  - Create and run [YAML-defined rules](#creating-and-using-yaml-defined-rules-file).
-- Run your [local rules simultaneously with Semgrep Registry rules](#running-multiple-rules-simultaneously). 
+  - One-off [ephemeral rules](#ephemeral-rules) used in the command line.
+  - The [YAML-defined rules](#creating-and-using-yaml-defined-rules-file) in a file.
+- Combination of [local rules with Semgrep Registry rules](#running-multiple-rules-simultaneously). 
 
-You can run all rules on your local code or continuously in your Source Code Management (SCM) service (such as GitHub or GitLab) with Semgrep in CI.
+You can run all rules on your code locally or continuously in your Source Code Management (SCM) service (such as GitHub or GitLab) with Semgrep in CI. For more information, see [Semgrep CI overview](semgrep-ci/overview.md).
 
-The following diagram displays the current number of Semgrep rules:
-<div className="lang-container" style={{marginBottom: '20px'}}>
-  <iframe width="900" height="400" frameBorder="0" src="https://dashboard.semgrep.dev/metric/semgrep-rules.num/graph"></iframe>
-</div>
-
-## Running Semgrep Registry rules
+## Running Semgrep Registry rules locally
 
-This section explains how to run [Semgrep Registry](https://semgrep.dev/explore) rules locally. See the following examples of how you may test Semgrep rules:
-
-1. Test how Semgrep automatically surveys languages and frameworks and run recommended rules for your source code:
-    ```sh
-    semgrep --config=auto path/to/src
-    ```
-2. Explore the [Semgrep Registry](https://semgrep.dev/explore) and choose a rule.
-3. On the page of the rule, click **Run Locally**.
-4. Copy the code for local install, and then add the path to the source code you want to check:
-    <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" <span className="placeholder">PATH/TO/SRC</span></code></pre>
-5. Optional: Run registry rules simultaneously with local rules:
-   <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" --config=<span className="placeholder">PATH/TO/YML PATH/TO/SRC</span></code></pre>
+Test how Semgrep automatically surveys languages and frameworks and run recommended rules for your source code (substitute the `PATH/TO/SRC` with path to your source code):
+<pre class="language-bash"><code>semgrep --config=auto <span className="placeholder">PATH/TO/SRC</span></code></pre>
 
-:::note
+:::info
 * `--config auto` sends your repository's project URL to [Semgrep Registry](https://semgrep.dev/r) to find rules configured for your repository and as a key for cached rule recommendations.
-* When Semgrep Registry is used, [usage metrics](../metrics) are collected by default.
```

Comment:
@brendongo suggests on slack that we are still collecting these metrics. Are the release notes wrong or referring to something else?

Reply:
Yes we are still collecting these metrics. Keep the line in please!

Reply:
Returned yesterday already. GitHub keeps it here because of the comment, it is not in source actually.